src/Microsoft.ML.Transforms/GroupTransform.cs

```diff
@@ -180,35 +180,42 @@ public override RowCursor[] GetRowCursorSet(Func<int, bool> predicate, int n, Ra
         }
 
         /// <summary>
+        /// The output schema collects columns used to group columns (aka group-key columns) columns being grouped from the source scheam.
```

Comment:
>scheam [](start = 131, length = 6)

tyop

Reply:
Nice catch.

---
In reply to: [245808223](https://github.com/dotnet/machinelearning/pull/2060#discussion_r245808223) [](ancestors = 245808223)